etc/docker/Dockerfile

```diff
@@ -81,13 +81,17 @@ ENV OTBR_OT_BACKBONE_CI_DEPS curl lcov wget build-essential
 ENV OTBR_NORELEASE_DEPS \
   cpputest-dev
 
+# Required for testing
+ENV OTBR_TEST_DEPS python3
```

Comment:
seems unrelated change?

Reply:
Seems python3 is installed along with bind9. So tests using python3 cmd will fail when DNS64=0.

Reply:
Maybe we can add python3 to OTBR_DOCKER_REQS?

Reply:
Update. Thanks!